qemu/tests/thin_provisioning_image.py

```diff
@@ -0,0 +1,165 @@
+import logging
+import time
+import os
+import re
+from autotest.client.shared import error
+from autotest.client.shared import utils
+from autotest.client.shared import data_dir as auto_data_dir
+from virttest import env_process
+from virttest import data_dir
+from virttest import gluster
+from virttest import qemu_storage
+from virttest import utils_disk
+from qemu.tests import thin_provisioning
+from avocado.utils import process
+from avocado.core import exceptions
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+      'thin-provisioning' functions test using local or gluster image:
+      1) boot up guest with local/gluster disk
+      2) get the block num stat
+      3) dd a file
+      4) get the block num stat
+      5) delete the file
+      6) fstrim
+      7) get the block num stat
+      8) check the block num stat
+
+      :param test: QEMU test object
+      :param params: Dictionary with the test parameters
+      :param env: Dictionary with test environment.
+      """
+
+    def count_block_num(image_disk):
+        """
+        Get the block num of the disk,when i create a large file in guest,the content needs
+        time to be written to the host disk,so when check using stat, you will find the block
+        number changes all the time for a period of time, this is related with the host env,
+        some machines is much faster,and some machines is slow, i test 3 host machines and finally
+        find when i set the wait_time to 12 seconds,it works fine..
```

Comment:
Please add the explanation to "wait_time = 12".
Here should be description about function().